src/main/java/org/openrsc/editor/Brush.java

```diff
@@ -1,4 +1,4 @@
-package com;
+package org.openrsc.editor;
```

Comment:
Biggest changes here. Renamed all variables for the menu and split the menus into their own classes. Injected button actions into the menus